huxley/www/js/components/AdvisorAssignmentsView.js

```diff
@@ -195,6 +193,14 @@ var AdvisorAssignmentsView = React.createClass({
     );
   },
 
+  _makeMap: function(objectList) {
```

Comment:
I think it's much more likely that we want the countries and committees as a lookup table than a list. So, I think they should be stored as an object in the Store (i.e, this list-to-map conversion happens when the countries/committees are fetched).

This will also save us calling `_makeMap` all over the place.